.github/workflows/test-client-libclient-py.yml

```diff
@@ -23,7 +27,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install tox
-          tox -e flake8,mypy --notest
+          tox -e flake8,mypy --notest --skip-pkg-install
```

Comment:
I assumed the following uses!
ref: https://tox.wiki/en/legacy/changelog.html#id121

Reply:
@mdonaka 

If `--skip-pkg-install` option is not added, the step which installs `quickmpc` (libc-py) package will be failed.
logs: https://github.com/acompany-develop/QuickMPC/actions/runs/4101749786/jobs/7073881278

After https://github.com/acompany-develop/QuickMPC/pull/103, our tests using `tox` needs `--skip-pkg-install` option because

- tox creates archive file using under `packages/client/libclient-py/`.
- tox runs install quickmpc with archive file
- tox cannot resolve path which is written `packages/client/libclient-py/pyproject.toml`

```console
dev@host:/path/to/libc-py$ tox -e mypy --notest
...
mypy: install_package> python -I -m pip install --force-reinstall --no-deps /path/to/QuickMPC/packages/client/libclient-py/.tox/.tmp/package/11/quickmpc-0.2.4.dev1+g3a76339.d20230206.tar.gz
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
...
      FileNotFoundError: [Errno 2] No such file or directory: '../../../packages/client/libclient-py/quickmpc/_version.py'
```

`tox -e flake8,mypy --notest --skip-pkg-install` command will create flake8 and mypy environments only with following configuration.

- flake8
  https://github.com/acompany-develop/QuickMPC/blob/3a57ff4881d7ed24bb120c719108198873b7cd23/packages/client/libclient-py/tox.ini#L10-L11
- mypy
  https://github.com/acompany-develop/QuickMPC/blob/3a57ff4881d7ed24bb120c719108198873b7cd23/packages/client/libclient-py/tox.ini#L15-L20

```console
dev@host:/path/to/libc-py$ rm -rf .tox
dev@host:/path/to/libc-py$ tox -e mypy --notest --skip-pkg-install
mypy: install_deps> python -I -m pip install mypy types-protobuf types-tabulate types-tqdm
mypy: skip building and installing the package
  mypy: OK (2.82 seconds)
  congratulations :) (3.01 seconds)
```

Reply:
I've been looking for a good solution for this problem, but haven't been able to find one.

Reply:
I see... That's tricky...
I understand, so OK! Thank you!